sdk/core/System.ClientModel/src/Message/PipelineResponseHeaders.cs

```diff
@@ -6,12 +6,31 @@
 
 namespace System.ClientModel.Primitives;
 
+/// <summary>
+/// A collection of HTTP response headers and their values.
+/// </summary>
 public abstract class PipelineResponseHeaders : IEnumerable<KeyValuePair<string, string>>
 {
+    /// <summary>
+    /// Attempts to retrieve the value associated with the specified header name.
+    /// </summary>
+    /// <param name="name">The name of the header to retrieve.</param>
+    /// <param name="value">The specified header value.</param>
+    /// <returns>true if the specified header name and value are stored in the collection; otherwise false.</returns>
```

Comment:
```suggestion
    /// <returns><c>true</c> if the specified header name and value are stored in the collection; otherwise <c>false</c>.</returns>
```